scripts/ci_checks/linux-checks/check-version.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+
+tagname="$(git tag -l | tail -n1)"
+
+header_tagname="v$(python3 scripts/scons_helpers/parse-version.py)"
+
+if [[ "$tagname" != "$header_tagname" ]]; then
+  echo
+  echo "version check FAILED" >&2
+  echo "the git tag name and version declared in version.h is not the same" >&2
+  echo "${tagname} git version tag" >&2
+  echo "${header_tagname} header version tag" >&2
+  echo -e "\n" 
+  exit 1
+fi
+
+echo "version check SUCCEEDED"
+echo -e "\n"
```

Comment:
Just a plain `echo` will be enough here (if you want to leave a single new line).

P.S.: Also, I started to think about is it OK to write a failure-text to stderr and a succeed-text to stdout. I suppose this script is not going to be used in a pipe where its output would be parsed by some other program. If so, it's completely fine to use different streams here (as you did in this pull request).